### DISCUSSION TITLE
Copilot not working on VS2022

### DISCUSSION THREAD

**Discussion by lbosquett** [community/community#33729] [Copilot Conversations]
Hi,

I'm using Copilot in Visual Studio 2022 (Community), but is not working, the suggestion keeps loading but never get any results nor any code completion, is working fine in VS Code (pretty fast indeed), but not on Visual Studio Community 2022.

![image](https://user-images.githubusercontent.com/46539053/191571608-5e6e189a-5e74-4c28-9d48-6e27304b8554.png)

Any ideas how to solve this?

Visual Studio: Community 2022 version 17.3.376.3011
Copilot version: 1.44.0.0
Copilot agent: 1.44.6735
Installed: 20/09/2022

**Comment by iamcjwilliams** [comment]
@lbosquett Please try the just-released Copilot 1.46.0.1 VS extension which contains various improvements
